### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by przemkow**
Update readme: 
* Reorganize documentation to make it easier to read 
* Fix typos
* Improve formatting (anchor should work also for isoDuration methods)

Preview: https://github.com/musement/iso-duration/blob/update-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,39 +61,17 @@ console.log(duration.humanize('en'))
...
 
-### i18n
-All languages which are expected to be available in `.humanize` method needs to be loaded using `isoDuration.setLocales`
-function. Currently, library provides support for languages listed under `/src/locales`.
+All languages which you want to use in `.humanize(lang)` method need to be initialised using `isoDuration.setLocales` function. Currently, the library provides support for languages listed under `/src/locales`.
+
+Languages not imported by your application will be removed by the module bundler (ex. webpack) during a build process using a tree-shaking mechanism. 
 
 ```js
 import { isoDuration, en, pl, it } from '@musement/iso-duration'
```
Comment (aureliome) [inline README.md:77]: Removing the signature from `Getting started`, you can add it here 😉 

```suggestion
import { isoDuration, en, pl, it } from '@musement/iso-duration';

// isoDuration.setLocales(locales, localesOptions);
```

In this way, the `localOptions` described below is clearer 😉

File: README.md [hunk 2]
```diff
@@ -100,19 +82,61 @@ isoDuration.setLocales({
...
   'en-GB': en,
   'en-US': en,  
+}, {
+  fallbackLocale: 'en'
 })
 ```
 
-#### Configuration
-`isoDuration.setLocales` can take an `option` object as second argument (optional):
+##### `LocalesOptions` parameter (optional)
```
Comment (aureliome) [inline README.md:90]:
```suggestion
#### `LocalesOptions` parameter (optional)
```

File: README.md [hunk 3]
```diff
@@ -100,19 +82,61 @@ isoDuration.setLocales({
...
+  fallbackLocale: string
+}
+```
+
+| Key              | Type   | Description                                                  | Default     |
+| ---------------- | ------ | ------------------------------------------------------------ | ----------- |
+| `fallbackLocale` | String | locale which needs to be when `IsoDuration.prototype.humanize(lang: string)` received incorrect parameter | `undefined` |
+
+
+## IsoDuration instance
```
Comment (aureliome) [inline README.md:103]: Like line 68, you can remove it imho 😉 

```suggestion

```

File: README.md [hunk 4]
```diff
@@ -126,26 +150,39 @@ console.log(isoDuration("P8T30M").parse())
...
+
 ```js
 console.log(isoDuration("P8T30M").humanize('en'))
 // 8 hours 30 minutes
 ```
 
-`HumanizeConfig` Object:
-```js
-{
+##### `HumanizeConfig` parameter (optional):
```
Comment (aureliome) [inline README.md:172]:
```suggestion
#### `HumanizeConfig` parameter (optional):
```

File: README.md [hunk 5]
```diff
@@ -160,27 +197,33 @@ console.log(
...
+ returns normalized IsoDuration object:
+
 ```js
 console.log(isoDuration("PT90M").normalize().toString())
 // PT1H30M
 ```
 
-This method takes an optional argument `date` which defines start of duration. It's used to correctly normalize number of days basing on corresponding month's length.   
-If it's not present normalize function will use current date (`new Date()`) instead.
+This method takes an optional argument `date` argument which defines the start of the duration. It's used to correctly normalize the number of days basing on the corresponding month's length.
```
Comment (aureliome) [inline README.md:209]: Typo: you have repeated `argument`

**Comment by przemkow** [comment]
thanks a lot, @aureliome, I've added changes suggested by you in the code review!

**Review by aureliome — APPROVED** [review]

**Pull request merged**